docs/index.asciidoc

```diff
@@ -48,9 +52,8 @@ output {
 }
 -----
 <1> `%{[@metadata][beat]}` sets the first part of the index name to the value
-of the `beat` metadata field and `%{[@metadata][version]}` sets the second part to
-the {plugin-uc} version. For example:
-metricbeat-7.4.0.
+of the metadata field and `%{[@metadata][version]}` sets the second part to
+the {plugin-singular} version. For example: metricbeat-7.4.0.
```

Comment:
@andsel Should this reflect the beats version (instead of the stack version)?

Reply:
`[@metadata][version]` field read from the remote beats is the version of the remote beat and not of the overall stack

Reply:
Thanks for validating. I will fix.